x-pack/plugins/apm/server/routes/services/annotations/index.ts

```diff
@@ -41,6 +45,14 @@ export async function getServiceAnnotations({
     searchAggregatedTransactions,
     start,
     end,
+  }).catch((error) => {
+    if (
+      error instanceof WrappedElasticsearchClientError &&
+      error.originalError instanceof errors.RequestAbortedError
+    ) {
+      return [];
+    }
+    throw error;
```

Comment:
Also, can you write a test first that confirms the bug?

Reply:
>Because this promise is not awaited if we use stored annotations 

In this case, another catch is added (line 71) to ignore the error thrown in the catch statement.

>Maybe what we need to do here is to only re-throw the error if we haven't returned any stored annotations before this error occurs.

I think that it what it does, since no stored annotations were found, the second catch to ignore the error won't be added. So if the exception is different than `RequestAbortedError` the error is going to be thrown up.

I'll see if I can simulate it on some test cases.

Reply:
I fixed what you commented here https://github.com/elastic/kibana/pull/135752/commits/e831c38820796e6713c77cc850548559d2b7ace2#diff-fba54d946f235ad77e5a30875a93d00c199b85199b09b3a1ee2160f3025fb75eR59, and added unit tests